SecureSubmit.php

```diff
@@ -1682,7 +1778,7 @@ function submit_payment() {
 
         // modal
         if (isset($_POST['cardholder_name'])) {
-            list($first, $middle, $last) = split(' ', $_POST['cardholder_name']);
+            list($first, $middle, $last) = preg_split ("/ /", $_POST['cardholder_name']);
```

Comment:
I don't have anything against regex, but I don't see any benefit to using it here. I typically only use it if there's a need.